client/src/components/Address/CreateAddress.js

```diff
@@ -32,9 +34,59 @@ const CreateAddress = () => {
   };
   let submitHandler;
   let id = location.pathname.slice(14);
-  if (location.pathname == "/user/address/null") {
+  if (location.pathname == newAddressUrl) {
     submitHandler = async (e) => {
       e.preventDefault();
+      if (name == null) {
+        setErrorMessage("Please enter name");
+        setTimeout(() => {
+          setErrorMessage(null);
+        }, 6000);
+        return;
+      }
+      if (email == null) {
+        setErrorMessage("Please enter email");
+        setTimeout(() => {
+          setErrorMessage(null);
+        }, 6000);
+        return;
+      }
+      if (phone == null) {
+        setErrorMessage("Please enter phone");
+        setTimeout(() => {
+          setErrorMessage(null);
```

Comment:
there are too many if conditions, can we do something to reduce that? or can we refactor this better? just asking

Reply:
I thought of keeping all the conditions in one if using or operator,but messages are different so kept all separately.